crates/codegen/parser/runtime/src/templates/language.rs.jinja2

```diff
@@ -219,11 +267,6 @@ impl Language {
         return Self::SUPPORTED_VERSIONS.iter().map(|v| v.to_string()).collect();
     }
 
-    #[napi(js_name = "scan", ts_return_type = "kinds.TokenKind | null", catch_unwind)]
```

Comment:
I should probably separate a PR that gets rid of the `scan` API, as outlined in #640

Reply:
+1 .. if that means we no longer have to expose `ScannedToken` or similar APIs.

Reply:
Separated this as https://github.com/NomicFoundation/slang/pull/728